src/Avalonia.X11/X11Clipboard.cs

```diff
@@ -52,7 +52,7 @@ Encoding GetStringEncoding(IntPtr atom)
                         : null;
         }
         
-        private unsafe void OnEvent(XEvent ev)
+        private unsafe void OnEvent(ref XEvent ev)
```

Comment:
Would it be better to use `in` rather than `ref` here (and in the other places this change was made) given that `ev` won't be changed?

Reply:
I *suspect* that some APIs do actually mutate the struct.